src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
I'm not clear why do we need this file?

Reply:
Should we use it in CI?

Reply:
This file is a part of JEST. Usd if you need a global setup before running. it will be automatically executed before running your tests. 
`import '@testing-library/jest-dom';` in this file make [App.test.js](https://github.com/skbu3u/web-interface/pull/3/files/8b14015696b9b21de08418d4bcd873f48efc5251#diff-1270de3bbb227435f61afc0333dccf3e1bcde4a082bf31a61acf489ae9bb3200) work with DOM.
I dont know should we use it in CI right now , i need read more informiation from official documentation of JEST.